sgm/modules/attention.py

```diff
@@ -1,12 +1,13 @@
 import math
-from inspect import isfunction
 from typing import Any, Optional
 
 import torch
 import torch.nn.functional as F
 from einops import rearrange, repeat
 from packaging import version
 from torch import nn
+from sgm.util import exists, default
```

Comment:
import as `from ..util import ... `

Reply:
This repo is currently mixing and matching both import styles. If you merge #52, I can enable a Ruff lint that forces `sgm` imports to always be relative.